tests/test_dice_interface/test_dice_genetic.py

```diff
@@ -177,7 +177,43 @@ def test_predict_custom(self, desired_class, sample_custom_query_2, mocker):
         mocker.patch('dice_ml.model_interfaces.base_model.BaseModel.get_output', return_value=[[0, 0.5, 0.5]])
         custom_preds = self.exp._predict_fn_custom(sample_custom_query_2, desired_class)
         assert custom_preds[0] == desired_class
-
+    
+    # Testing if the shapes of the predictions are correct for multiclass classification
+    @pytest.mark.parametrize(("desired_class", "method"), [(1, "genetic")])
+    def test_multiclass_nn(self, desired_class, method):
+        pytest.importorskip("torch")
+        pytest.importorskip("sklearn")
+
+        backend = "PYT"
+        dataset = helpers.load_custom_testing_dataset_multiclass()
+
+        # Transform the categorical data to number to test the neural network
+        import sklearn 
```

Comment:
Please move all imports to top of the file with other imports.

Reply:
I imported it within the function to avoid changing the global scope, but I will change this too, thanks.

Reply:
yeah, good to change. It will also resolve the linting error that github is showing:
> Run isort . -c
ERROR: /home/runner/work/DiCE/DiCE/tests/test_dice_interface/test_dice_genetic.py Imports are incorrectly sorted and/or formatted.

Reply:
After the changes in the commit I did yesterday, it seems that there was still an error in the linting. I run isort locally and it removes a whitespace, let's see if it was the problem.

Reply:
this is due to flake8. see my comment below.